Documents/TechnicalSpecifications/technicalSpecifications.md

```diff
@@ -38,15 +75,18 @@ In the team, two different types of computer are used:
 | Computer | OS | Number used |
 | -- | -- | -- |
 | MacBook Air M3 | MacOS | 4 |
-| ThinkBook 14 G4 | Windows 11 | 2 |
+| ThinkBook 14 G4 | Windows 11 | 3 |
```

Comment:
We have a 8th computer but this one is on linux (I'm not sure if it's also a Thinkbook 14, I don't think so)